### PR TITLE
Fix links in Operations reference

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by epugh — DISMISSED** [review]
Vale burped on one formatting with `//`.   Does vale prevent the `/api-reference/index-apis/` style link?

File: _benchmark/reference/workloads/operations.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ If `detailed-results` is `true`, the following metadata is returned:
 ## create-index
 <!-- vale on -->
 
-The `create-index` operation runs the [Create Index API](/api-reference/index-apis/create-index/). It supports the following two modes of index creation:
+The `create-index` operation runs the [Create Index API]({{site.url}}{{site.baseurl}}//api-reference/index-apis/create-index/). It supports the following two modes of index creation:
```
Comment (Naarcha-AWS) [inline _benchmark/reference/workloads/operations.md:85]:
```suggestion
The `create-index` operation runs the [Create Index API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/). It supports the following two modes of index creation:
```

**Comment by Naarcha-AWS** [comment]
> Vale burped on one formatting with `//`. Does vale prevent the `/api-reference/index-apis/` style link?

It does since double slashes break the link. Fixed.

File: _benchmark/reference/workloads/operations.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ The `create-index` operation returns the following metadata:
 ## delete-index
 <!-- vale on -->
 
-The `delete-index` operation runs the [Delete Index API](api-reference/index-apis/delete-index/). Like with the [`create-index`](#create-index) operation, you can delete all indexes found in the `indices` section of the workload or delete one or more indexes based on the string passed in the `index` setting.
+The `delete-index` operation runs the [Delete Index API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/delete-index/). Like with the [`create-index`](#create-index) operation, you can delete all indexes found in the `indices` section of the workload or delete one or more indexes based on the string passed in the `index` setting.
```
Comment (natebower) [inline _benchmark/reference/workloads/operations.md:160]:
```suggestion
The `delete-index` operation runs the [Delete Index API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/delete-index/). As with the [`create-index`](#create-index) operation, you can delete all indexes found in the `indices` section of the workload or delete one or more indexes based on the string passed in the `index` setting.
```

File: _benchmark/reference/workloads/operations.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ The `delete-index` operation returns the following metadata:
 ## cluster-health
 <!-- vale on -->
 
-The `cluster-health` operation runs the [Cluster Health API](api-reference/cluster-api/cluster-health/), which checks the cluster health status and returns the expected status according to the parameters set for `request-params`. If an unexpected cluster health status is returned, the operation reports a failure. You can use the `--on-error` option in the OpenSearch Benchmark `execute-test` command to control how OpenSearch Benchmark behaves when the health check fails.
+The `cluster-health` operation runs the [Cluster Health API]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-health/), which checks the cluster health status and returns the expected status according to the parameters set for `request-params`. If an unexpected cluster health status is returned, the operation reports a failure. You can use the `--on-error` option in the OpenSearch Benchmark `execute-test` command to control how OpenSearch Benchmark behaves when the health check fails.
```
Comment (natebower) [inline _benchmark/reference/workloads/operations.md:218]:
```suggestion
The `cluster-health` operation runs the [Cluster Health API]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-health/), which checks the cluster health status and returns the expected status according to the parameters set for `request-params`. If an unexpected cluster health status is returned, then the operation reports a failure. You can use the `--on-error` option in the OpenSearch Benchmark `execute-test` command to control how OpenSearch Benchmark behaves when the health check fails.
```

**Pull request merged**